test/image.test.js

```diff
@@ -150,4 +150,56 @@ describe('Image', function () {
     image.src = new Buffer(0);
     image.src = new Buffer('');
   });
+
+  it('should unbind Image#onload', function() {
+    var img = new Image
+      , onloadCalled = 0;
+
+    img.onload = function() {
+      ++onloadCalled;
```

Comment:
If you wouldn't mind, please change this to `onloadCalled += 1` because of the reasons we discussed before... Thanks